libs/constrained-generators/src/Constrained/GenT.hs

```diff
@@ -200,13 +203,13 @@ suchThatT g p = do
   mode <- getMode
   let (n, cont) = case mode of
         Strict -> (100, fatalError)
-        Loose -> (1 :: Int, genError) -- TODO: Maybe 1 is not the right number here!
-  go n cont
-  where
-    go 0 cont = cont ["Ran out of tries on suchThatT"]
-    go n cont = do
-      a <- g
-      if p a then pure a else scaleT (+ 1) $ go (n - 1) cont
+        Loose -> (100 :: Int, genError) -- TODO: Maybe 1 is not the right number here!
```

Comment:
Why is it necessary to change this to `100`? It will slow a lot of things down by about a factor 100.

Reply:
I have examples, run in the Strict mode that needed multiple tries to succeed. I don't think this is called alot, in strict mode, (but it is called), and I didn't notice any slowdown.